documentation/asciidoc/computers/raspberry-pi/boot-http.adoc

```diff
@@ -0,0 +1,50 @@
+== HTTP Boot
+
+The network install feature uses HTTP over ethernet to boot the device into the xref:getting-started.adoc#using-raspberry-pi-imager[Raspberry Pi Imager] application.
+
+In addition to network install you can explicitly boot your device with files downloaded via HTTP with xref:raspberry-pi.adoc#BOOT_ORDER[boot mode] `7`. You can still use this even if xref:raspberry-pi.adoc#NET_INSTALL_ENABLED[network install / keyboard detection on boot is disabled].
```

Comment:
The `/` in "network install / keyboard detection on boot" is a bit weird, and makes it sound like those are two separate things?

Reply:
ok - removed 3 instances of this